app/javascript/controllers/admin_days_limit_controller.js

```diff
@@ -0,0 +1,9 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
```

Comment:
Can you name this controller more generically, e.g., `form_submit_controller.js`?